src/parse_data.py

```diff
@@ -0,0 +1,85 @@
+import collections
+import csv
+import os
+
+# local configuration
+data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data')
+data_path_remote = os.path.join(data_path, 'remote')
+data_path_static = os.path.join(data_path, 'static')
+
+
+def load_csv_file(loc_type, name):
+	"""
+	Loads CSV data into an array of dictionaries.
+
+	Params:
+		loc_type (str): Either 'remote' or 'static'. Static is assumed if an incorrect value is provided.
+		name (str): The combined name and filetype of the file to load.
+
+	Returns:
+		An array of dictionaries containing the data read from the file.
+
+	Note:
+		The CSV file must contain a header row.
+
+	Warning:
+		Behavior when loading data from a location that does not contain valid CSV data is unspecified.
+	"""
+	path = data_path_remote if (loc_type == 'remote') else data_path_static
+
+	# TODO: Properly deal with files that don't exist
+	with open(os.path.join(path, name), 'r') as f:
+		reader = csv.DictReader(f)
+
+		return list(reader)
+
+
+def publisherify_data(base_info, summary_stats):
+	"""
+	Converts data to be in a format that allows direct querying by publisher.
+
+	Params:
+		base_info (array of dict): An array of dictionaries containing the base info.
+		summary_stats (array of dict): An array of dictionaries containing the summary stats.
+
```

Comment:
I would suggest being more precise about what type of array is expected in the input params - i.e. a list or set? I don't think the function would work if an associative array was used.